modern/base/Dockerfile

```diff
@@ -93,11 +95,13 @@ RUN apt-get -qq update \
     && rm -rf /tmp/* \
     && runuser -u conan -- printf 'eval "$(pyenv init -)"\n' >> /home/conan/.bashrc \
     && runuser -u conan -- printf 'eval "$(pyenv virtualenv-init -)"\n' >> ~/.bashrc \
-    && chown -R conan:1001 /opt/pyenv
+    && chown -R conan:1001 /opt/pyenv \
+    && chown -R conan:1001 /opt/conan
 
 USER conan
 WORKDIR /home/conan
 ENV CONAN_USER_HOME=/home/conan
 
 RUN pip install conan==${CONAN_VERSION} \
-    && conan config set general.revisions_enabled=1
+    && conan config set general.revisions_enabled=1 \
+    && CONAN_USER_HOME=/opt/conan CONAN_REVISIONS_ENABLED=1 conan install -r conancenter -g deploy -if /opt/conan -l /opt/conan/conan.lock binutils/2.37@
```

Comment:
Isn't `CONAN_REVISIONS_ENABLED=1` redundant given that one line earlier revisions are enabled globally?

Reply:
No, because there is a trick here:
At line 106, it configures revisions enabled at `/home/conan/.conan/conan.conf` which will be used by any user. 
At line 107 we create a new conan cache at `/opt/conan/.conan` which is temporary only to install those packages.

Why keeping them separated? I want to keep Conan client fresher as possible when running at first time. When we install bintuils, Conan client also populates other files like, remotes, settings, .... For CCI it should be okay, but maybe if you are using it, you don't want an "used" environment.

You comment makes me think we should use `CONAN_REVISIONS_ENABLED` by default, on first lines of this Dockerfile.